andela/ah-orcas-frontend
# #161254625 like dislike article

#### What does this PR do?
- enables a user to like or dislike an article. 

#### Description of Task to be completed?
- Lets appreciate those authors who go out of their way to create good content with a Like and encourage those authors who do not meet our expectations with a dislike.

#### How should this be manually tested?
- git clone this repo
```
$ git clone https://github.com/andela/ah-orcas-frontend
```
- Navigate to this branch
```
git checkout ft-like-dislike-#161254625
```
- Install the node modules
```
$ npm install
```
- Start the server
```
$ npm start
```
- It should open your browser to localhost port 3000.
- Log in via the login page then navigate to a single article.
- You should find two icons fro like and dislike. 
- Try to use them and see if they work as expected.

#### Any background context you want to provide?
- This was only to add the functionality for like and dislike.

#### What are the relevant pivotal tracker stories?
[#161254625](https://www.pivotaltracker.com/story/show/161254625)
#### Screenshots (if appropriate)
- How they look.
<img width="90" alt="screenshot 2018-12-18 at 19 14 29" src="https://user-images.githubusercontent.com/26874561/50168844-64a49580-02fd-11e9-9ee3-6546e9603cba.png">
- As you wait for dislike to process.
<img width="92" alt="screenshot 2018-12-18 at 19 15 34" src="https://user-images.githubusercontent.com/26874561/50168838-640bff00-02fd-11e9-8a9a-67cddbc9af63.png">
- As you wait for like to process.
<img width="80" alt="screenshot 2018-12-18 at 19 15 09" src="https://user-images.githubusercontent.com/26874561/50168842-64a49580-02fd-11e9-8ee8-683dded8bf3c.png">
- How they look with an article
<img width="1184" alt="screenshot 2018-12-20 at 15 47 24" src="https://user-images.githubusercontent.com/26874561/50285893-a6088280-046e-11e9-84fa-92e2b51ce40d.png">



#### Checklist:
